### PR TITLE
Enable px4io

### PR DESCRIPTION
**Pull request by jlaitine**
This enables PX4IO on our saluki-nxp93 HW

### CODE REVIEW

File: platforms/nuttx/src/px4/nxp/imx9/version/board_mcu_version.c [hunk 1]
```diff
@@ -70,8 +70,8 @@
 #  define min(a,b) (((a) < (b)) ? (a) : (b))
 #endif
 
-static unsigned hw_version = 0;
-static unsigned hw_revision = 0;
+static unsigned hw_version = BOARD_DEFAULT_VER;
```
Comment (pussuw) [inline platforms/nuttx/src/px4/nxp/imx9/version/board_mcu_version.c:73]: Unrelated change

**Review by pussuw — APPROVED** [review]
LGTM

File: platforms/nuttx/src/px4/nxp/imx9/version/board_mcu_version.c (hunk 1 above)
Comment (jlaitine) [reply platforms/nuttx/src/px4/nxp/imx9/version/board_mcu_version.c:73]: It's kind of related. The version numbers are used in manifest parsing (selecting the correct options for the HW in guestion). The manifest parsing /version numbering could have been put into an own PR of course. But it was also needed in order to enable PX4IO, existence of which is read from the manifest (based on hw version)....

So I just decided to mash everything in this same PR, sorry for that and thanks for the comment :)
